net/client.go

```diff
@@ -24,6 +24,9 @@ func makeTimeoutHTTPClient(dialTimeout time.Duration, fullTimeout time.Duration,
 			TLSHandshakeTimeout: dialTimeout,
 			MaxIdleConnsPerHost: maxIdle,
 			IdleConnTimeout:     15 * time.Second,
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
+			},
```

Comment:
Should this change really be tracked?  :see_no_evil:

Reply:
oops